share/setup.nsi.in

```diff
@@ -21,7 +21,7 @@ SetCompressor /SOLID lzma
 !define MUI_STARTMENUPAGE_DEFAULTFOLDER "@PACKAGE_NAME@"
 !define MUI_FINISHPAGE_RUN "$WINDIR\explorer.exe"
 !define MUI_FINISHPAGE_RUN_PARAMETERS $INSTDIR\@BITCOIN_GUI_NAME@@EXEEXT@
-!define MUI_UNICON "${NSISDIR}\Contrib\Graphics\Icons\modern-uninstall.ico"
+!define MUI_UNICON "@abs_top_srcdir@/share/pixmaps/bitcoin-black.ico"
```

Comment:
should be `\` not `/`

Reply:
I don't think so. It was backslashes because that was NSIS directory which I think only appears when installing on Windows.
The pixmaps/icons are in the source folder. Take a look at how bitcoin-qt.exe icon is defined:
`!define MUI_ICON "@abs_top_srcdir@/share/pixmaps/bitcoin.ico"`